paasta_tools/tron_tools.py

```diff
@@ -527,25 +527,27 @@ def load_tron_instance_config(
 
 
 def load_tron_yaml(service: str, cluster: str, soa_dir: str) -> Dict[str, Any]:
-    tronfig_folder = get_tronfig_folder(soa_dir=soa_dir, cluster=cluster)
     config = service_configuration_lib.read_extra_service_information(
         service_name=service,
         extra_info=f'tron-{cluster}',
         soa_dir=soa_dir,
     )
-    if not config:
-        config = service_configuration_lib._read_yaml_file(os.path.join(tronfig_folder, f"{service}.yaml"))
     if not config:
         raise NoConfigurationForServiceError('No Tron configuration found for service %s' % service)
     return config
 
 
+def extract_jobs_from_tron_yaml(config):
+    config = {key: value for key, value in config.items() if not key.startswith('_')}  # filter templates
+    if 'jobs' in config and config.get('jobs') is None:
```

Comment:
i  would make this path more explicitly "deprecated":

    if 'jobs' in config:
      log.warn("jobs key in per-service tronfig is deprecated, please de-dent")
      return config.get('jobs', {})
    else:
      return config

Reply:
I would rather not. 
I see those deprecation warnings for those users who we can't control.

I control all of the configs at yelp, so after this change I'll remove the `jobs` key everywhere. No need for anyone else to see it. Plus, nobody would see this anyway, it ends up on the tron server.

Reply:
you're right, users won't see this. i was thinking about somebody naming their job "jobs" - we  should somehow warn this will not work correctly.

Reply:
Haha. That is a funny edge case, but it will soon be not important.

This code is only for the transition period. I plan to:
* ship this
* de-indent all the things and remove the jobs word in soa-configs
* update this code to just assume everything in soa-configs are jobs

Reply:
Ok, ship away!